nas/auth.go

```diff
@@ -341,14 +352,24 @@ func svcloc(fields map[string]string) map[string]string {
 }
 
 func handleProfanity(fields map[string]string) map[string]string {
-	prwords := ""
-	wordCount := strings.Count(fields["words"], "\t") + 1
-	for i := 0; i < wordCount; i++ {
-		prwords += "0"
+	var prwords string
+	for _, word := range strings.Split(fields["words"], "\t") {
+		if isBadWord, _ := IsBadWord(word); isBadWord {
+			prwords += "1"
+		} else {
+			prwords += "0"
+		}
+	}
```

Comment:
Can `prwords` be a boolean value that once a bad word is encountered, it is set to true and the loop is broken out of?

Reply:
No, the client expects a value for every word